modules/benchmark/R/calc_benchmark.R

```diff
@@ -172,7 +198,12 @@ calc_benchmark <- function(settings, bety) {
     gridExtra::grid.table(do.call(rbind, results.list))
     dev.off()
     
-    names(dat.list) <- var.list
+    var.names <- c()
+    for(k in seq_along(dat.list)){
+      var.names <- c(var.names,unlist(strsplit(names(dat.list[[k]])[grep("[.]m", names(dat.list[[k]]))],"[.]"))[1]) # This is horrifying. Sorry future self. 
```

Comment:
That is my fault - it was an AGU hack and shouldn't have made it to the pull request. I'll rewrite it.